onnxruntime/python/tools/quantization/onnx_quantizer.py

```diff
@@ -793,6 +809,7 @@ def calculate_quantization_params(self):
             self.tensors_range[node.input[0]] = self.tensors_range[node.output[0]]
 
         quantization_params = {}
+        symmetric = 'Symmetric' not in self.extra_options or self.extra_options['Symmetric']
```

Comment:
>Symmetric [](http://example.com/codeflow?start=21&length=9)

Maybe a name 'WeightSymmetric' is a better name

Reply:
Updated.